File: jecnaapi-java/src/examples/java/AttendanceExample.java
import me.tomasan7.jecnaapi.data.attendance.AttendancesPage;
import me.tomasan7.jecnaapi.java.JecnaClientJavaWrapper;

import java.time.LocalDate;
import java.util.Set;


public class AttendanceExample
{
    public static void main(String[] args)
    {
        /* Metody join() používáme aby jsme z asynchroního běhu "udělali" synchroní; pouze v příkladech */

        JecnaClientJavaWrapper client = new JecnaClientJavaWrapper();

        client.login("user", "password").join();

        /* Lze specifikovat období v argumentu */
        AttendancesPage attendancesPage = client.getAttendancePage().join();

        /* Všechny dny, pro které existují data */
        Set<LocalDate> attendances = attendancesPage.getDays();

        /* Všechny příchody a odchody (v daném půlroce) */
        for (LocalDate date : attendances)
            System.out.println(attendancesPage.get(date));
    }
}

File: src/examples/kotlin/AttendanceExample.kt
import kotlinx.coroutines.runBlocking
import me.tomasan7.jecnaapi.JecnaClient

fun main(): Unit = runBlocking {

    val client = JecnaClient()

    client.login("user", "password")

    /* Lze specifikovat období v argumentu */
    val attendancesPage = client.getAttendancesPage()

    /* Všechny dny, pro které existují data */
    val attendedDays = attendancesPage.days

    /* Všechny příchody a odchody (v daném půlroce) */
    for (day in attendedDays)
        println(attendancesPage[day])
}
